locale/fi.po

```diff
@@ -5205,6 +5227,10 @@ msgstr ""
 msgid "TOGGLE_PAUSE"
 msgstr "Pysäytä"
 
+#, fuzzy
```

Comment:
Could you redo the translations from scratch for this PR? I see this and the en.po file have a bunch of old translations added that will never be useful as the translation keys have never existed on the master branch.

Reply:
Also I see this has translation file conflicts so anyway redoing the translations shouldn't be harder than fixing those conflicts manually.